app/controllers/declaration/ConsignmentReferencesController.scala

```diff
@@ -21,8 +21,9 @@ import controllers.navigation.Navigator
 import forms.declaration.ConsignmentReferences
 import forms.declaration.ConsignmentReferences.form
 import javax.inject.Inject
+import models.DeclarationStatus.DeclarationStatus
 import models.requests.JourneyRequest
-import models.{DeclarationType, ExportsDeclaration, Mode}
+import models.{DeclarationStatus, DeclarationType, ExportsDeclaration, Mode}
```

Comment:
Those imports are not necessary after removing implementation from this file

Reply:
I know - forgot to tidy imports in my rush to update the PR.   Next commit!